File: 23-03-18/java/01-Min-Max.java
import java.util.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        int ar[] = new int[n];
        int sum = 0, mini = Integer.MAX_VALUE, maxi = 0;
        
        for(int i = 0; i < n; i++)
        {
            ar[i] = input.nextInt();
            sum += ar[i];
            if(ar[i] > maxi)
                maxi = ar[i];
            if(ar[i] < mini)
                mini = ar[i];
        }
        
        System.out.println( (sum-maxi) + " " + (sum-mini) );
        
    }
}

File: 23-03-18/c++/01-Min-Max.cpp
#include<bits/stdc++.h>

using namespace std;

int main()
{
    int n;
    cin >> n;
    int ar[n];
    int sum = 0, maxi = INT_MIN, mini = INT_MAX;
    
    for(int i = 0; i < n; i++)
    {
        cin >> ar[i];
        sum += ar[i];
        
        if(ar[i] < mini)
            mini = ar[i];
        else if(ar[i] > maxi)
            maxi = ar[i];
    }
    
    cout << (sum-maxi) << " " << (sum-mini) << endl;
}
